components/heyy/common/utils.mjs

```
@@ -0,0 +1,60 @@
+      throw new Error("Not an array");
+    }
+
+    return parsedValue;
+
+  } catch (e) {
+    throw new ConfigurationError("Make sure the custom expression contains a valid array object");
+  }
+}
+
+function isPhoneNumberValid(phoneNumber) {
+  const pattern = new RegExp("^\\+[1-9]{1}[0-9]{0,2}[2-9]{1}[0-9]{2}[2-9]{1}[0-9]{2}[0-9]{4}$");
```

Biome

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)